reference/env_vars.rst

```diff
@@ -1,15 +1,18 @@
 .. _env_vars:
 
 Environment variables
-=============================
+=====================
 
 These are the environment variables used to customize conan.
 
-Most of them can be set in the ``conan.conf`` configuration file (inside your ``<userhome>/.conan`` folder)
+Most of them can be set in the ``conan.conf`` configuration file (inside your ``<userhome>/.conan`` folder).
 
+Variables
```

Comment:
Why this new level? Seems uninformative.

Reply:
Just to get the headers highlight right (There is a CMake Related Variables after at the same highlight level).

Any suggestion for a more informative name?

Reply:
I think I prefer the current levels, that is too much of nesting, and I think it might put the last variables like CONAN_USER_HOME_SHORT, etc under ``CMake Related Variables``

Reply:
Oh Ok, now I see it.